VideoLocker/src/main/java/org/edx/mobile/services/EdxCookieManager.java

```diff
@@ -53,31 +53,6 @@ public void clearWebWiewCookie(Context context){
             }
         }
         authSessionCookieExpiration = -1;
-
-    }
-
-    public void clearWebViewCache(Context context){
-        try {
-            clearWebWiewCookie(context);
-
-            boolean success = context.deleteDatabase("webview.db");
-            logger.debug("delete webview.db result = " + success);
-            success =  context.deleteDatabase("webviewCache.db");
-            logger.debug("delete webviewCache.db result = " + success);
-        } catch (Exception e) {
```

Comment:
While clearing the `WebView` cache upon upgrades might not be necessary, that doesn't seem to what's triggering the `SQLiteException` that is being reported. That seems to be triggered by initializing `WebViewDatabaseClassic`, which is done by the `WebView` at some point. This code is only deleting the database, not performing any transactions, and furthermore it's already catching and discarding all exceptions.